src/main/java/com/crud/library/domain/BookDto.java

```diff
@@ -0,0 +1,63 @@
+package com.crud.library.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@NoArgsConstructor
+@Getter
```

Comment:
stosujesz w sumie dużo tego nieszczęsnego Lomboka to może użyj też annotacji `@Builder` i Ci po prostu stworzy buildery ;)